file-formats/evto/examples/z_aff_example.evto.json

```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example Event Object.",
+    "originalLanguage": "en"
+  },
+  "majorVersion": 1,
+  "minorVersion": 0,
+  "patchVersion": 4,
+  "sapObjectType": "Z_AFF_EXAMPLE"
```

Comment:
We usually suggest (in the meantime) to not introduce top-level fields. If you don't want to have a specific name for the section you could name it `general_information`.

See https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#writing-json-schema-with-abap-types

Reply:
Yes, I will incorporate this suggestion.